cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js

```diff
@@ -47,68 +47,70 @@ const visitPage = (service, pageType) => {
   });
 };
 
-Object.keys(config)
-  .filter(serviceFilter)
-  .forEach(service => {
-    Object.keys(config[service].pageTypes)
-      .filter(pageType => filterPageTypes(service, pageType))
-      .forEach(pageType => {
-        describeForEuOnly(
-          `Amp Cookie Banner Test for ${service} ${pageType}`,
-          () => {
-            beforeEach(() => {
-              visitPage(service, pageType);
-            });
+if (Cypress.env('APP_ENV') !== 'local') {
```

Comment:
Can we add some comments as to why we are disabling these tests on localhost please